src/assets/scss/dropdown.scss

```diff
@@ -9,9 +9,9 @@
   }
 
   .btn-selected {
-    background: $white;
+    background: $blue;
```

Comment:
This would be applied to the current dropdowns too. Let's add a conditional class for if side filters enabled.